### PR TITLE
shalinee | TEST | resolved conflict

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/settings/address-settings/address-settings.component.html [hunk 1]
```diff
@@ -116,37 +116,38 @@ <h1>No addresses found within given criteria.</h1>
...
-                                    as Default</p>
-                                <p (click)="handleSetDefaultAddress(address)"
-                                    class="selected-defualt mr-b1 cursor-pointer">
-                                    <i *ngIf="address.isDefault" class="icon-favorite-selected mr-r05"></i>
-                                    <i *ngIf="!address.isDefault" class="icon-no-selected mr-r05"></i>
-                                    Set as Default</p>
-                                <p (click)="showConfirmationModal(address)" class="cursor-pointer">
-                                    <i class="icon-unlink mr-r05"></i>Unlink Address</p>
+            <div *ngFor="let address of addresses; index as addressIndex">
+                <div class="branch-profile-address-content col-md-12 mb-15">
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/settings/address-settings/address-settings.component.html:120]: **branch-profile-address-content** , wrap by new div and changes the position of ngFOR

**Review by prateek-giddh — APPROVED** [review]

File: apps/web-giddh/src/app/settings/address-settings/address-settings.component.html [hunk 2]
```diff
@@ -116,37 +116,38 @@ <h1>No addresses found within given criteria.</h1>
...
+                    <div class="single-branch-detail mb-15">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/settings/address-settings/address-settings.component.html:121]: @shalinee-walkover replace class name = mb-15 by mr-b15

File: apps/web-giddh/src/app/settings/address-settings/address-settings.component.html [hunk 3]
```diff
@@ -116,37 +116,38 @@ <h1>No addresses found within given criteria.</h1>
...
+                                    <p class="no-selected mb-1 cursor-pointer" hidden><i
+                                            class="icon-no-selected mr-05"></i>Set
+                                        as Default</p>
+                                    <p (click)="handleSetDefaultAddress(address)"
+                                        class="selected-defualt mb-1 cursor-pointer">
+                                        <i *ngIf="address.isDefault" class="icon-favorite-selected mr-05"></i>
+                                        <i *ngIf="!address.isDefault" class="icon-no-selected mr-05"></i>
+                                        Set as Default</p>
+                                    <p (click)="showConfirmationModal(address)" class="cursor-pointer">
+                                        <i class="icon-unlink mr-05"></i>Unlink Address</p>
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/settings/address-settings/address-settings.component.html:149]: if this is for margin-right 5px then replace class name =mr-05 to mr-r05 @shalinee-walkover

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/218gyz

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**